common/test/com/commercetools/sunrise/framework/viewmodels/forms/QueryStringUtilsTest.java

```diff
@@ -101,4 +102,17 @@ public void extractsQueryString() throws Exception {
                 assertThat(extractQueryString(httpContext.request()))
                         .isEqualTo(queryString));
     }
+
+    @Test
+    public void extractsQueryStringIgnoringAParameter() throws Exception {
+
+        final String IGNORED_PARAM = "foo";
+        final Map<String, List<String>> queryString = someQueryString();
+
+        queryString.remove(IGNORED_PARAM);
```

Comment:
If you remove the parameter it won't be there for the test and it will always pass ;)